### ISSUE TITLE
Accessibility Audit: Keyboard focus indicator is not visible in high contrast mode

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#965]
**ENVIRONMENT**
Windows 10 | Firefox 94.0.2(64-bit)

**ISSUE**

Enable High Contrast Black Mode (Settings > Personalization > Colors > High Contrast Settings), and go to the [Listings](https://sites-public-tj3gg4i5eq-uc.a.run.app/listings) page. Open the Filter modal, using the keyboard tab key only. Navigate through the modal, and notice that a focus indicator is not visible around the controls. A clear focus indicator should be visible around the controls when the user navigates to them using tab key. This works when viewing the page regularly. I was also able to replicate this in Firefox on Macbook using High Contrast Mode.
